### PR TITLE
Scope css

### PR DESCRIPTION
**Pull request by cat2608**
Apply CSS Modules

### CODE REVIEW

File: src/comparison-table/components/Amount.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 
+import style from './style.less';
+
 const Amount = ({ currency, value, precision, symbol, ...props }) => (
   <strong {...props}>
     {symbol}
     <span>
-      {parseFloat(value).toFixed(precision || 2)} <span className="text-primary">{currency}</span>
+      {parseFloat(value).toFixed(precision || 2)} <span className={`${style['text-primary']}`}>{currency}</span>
```
Comment (keyvanakbary) [inline src/comparison-table/components/Amount.js]: Probably the quotes are not necessary?

File: src/comparison-table/components/ComparisonTable.js [hunk 2]
```diff
@@ -204,8 +211,8 @@ class ComparisonTable extends Component {
         </div>
         {
           !this.transferwiseIsTheCheapestProvider() &&
-            <div className="m-b-2">
-              <div className="alert alert-info" role="alert">
+            <div className={`${style['m-b-2']}`}>
```
Comment (keyvanakbary) [inline src/comparison-table/components/ComparisonTable.js]: Probably the quotes are not needed?

File: src/comparison-table/components/Provider.js [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
         value={rate}
         precision={4}
       />
     </td>
 
-    <td className="text-xs-right">
+    <td className={style['text-xs-right']}>
       <Amount
-        className="text-no-wrap text-primary"
+        className={`${style['text-primary']}`}
```
Comment (keyvanakbary) [inline src/comparison-table/components/Provider.js]: Probably the quotes are not needed?

File: src/comparison-table/components/atoms/ArrowIcon.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 
+import style from '../style.less';
+
 const Arrow = ({ iconPosition }) => (
-  <div className={`arrow-icon ${iconPosition}`}>
+  <div className={`${style['arrow-icon']} ${iconPosition ? style[iconPosition] : ''}`}>
```
Comment (daves125125) [inline src/comparison-table/components/atoms/ArrowIcon.js]: why the empty ternary for the icon position?

File: src/comparison-widget/components/atoms/Select.js [hunk 5]
```diff
@@ -34,27 +34,25 @@ class Select extends Component {
...
     this.props.changeAmount(value);
   }
 
   render () {
     const label = this.props.options[parseInt(this.props.amount, 10)];
+    const style = this.props.styleTable;
 
     return (
-      <div className={this.state.open ? 'Select open' : 'Select'}>
+      <div className={`Select  ${style.h2} ${this.state.open ? style.open : ''}`}>
```
Comment (daves125125) [inline src/comparison-widget/components/atoms/Select.js]: Will "select" not clash? or does this also get the hash appended to its class name?

File: package.json [hunk 6]
```diff
@@ -55,10 +56,12 @@
     "gulp": "^3.9.1",
     "gulp-merge-json": "^1.1.0",
     "html-webpack-plugin": "^2.30.1",
+    "identity-obj-proxy": "^3.0.0",
```
[the remark this answers is not in the document]
Comment (cat2608) [reply package.json]: Nowhere, mistake :/

File: src/comparison-widget/components/atoms/Select.js (hunk 5 above)
Comment (cat2608) [reply src/comparison-widget/components/atoms/Select.js]: This `Select` comes from custom style and we just need to import like `import '../style.css';`. This one will not clash.

File: src/comparison-table/components/atoms/ArrowIcon.js (hunk 4 above)
Comment (cat2608) [reply src/comparison-table/components/atoms/ArrowIcon.js]: When `iconPosition` the class name is `<div class="arrow-icon_1DyMKTwOgG up_1l5xbziTsG"></div>` and when is not passed is `<div class="arrow-icon_1DyMKTwOgG"></div>`

**Pull request merged**